src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_ITupleExpression.cs

```diff
@@ -0,0 +1,473 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.CSharp.Test.Utilities;
+using Microsoft.CodeAnalysis.Test.Utilities;
+using Roslyn.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.CSharp.UnitTests
+{
+    public partial class IOperationTests : SemanticModelTestBase
```

Comment:
>IOperationTests [](start = 25, length = 15)

Do we have a test that uses user-defined conversion on a tuple literal, which requires conversions for tuple elements, including user-defined ones?